c/busybox-1.22.0/realpath_true-no-overflow_true-valid-memsafety.c

```diff
@@ -283,8 +283,6 @@ signed int __main(signed int argc, char **argv)
     resolved_path=xmalloc_realpath(*argv);
     if(!(resolved_path == ((char *)NULL)))
     {
-      puts(resolved_path);
-      free((void *)resolved_path);
```

Comment:
I would prefer `busybox_sv_comp_impl.h` despite the unnecessary changes - hopefully we'll get rid of those preprocessed files in the not too distant future.